src/NHibernate/Type/BooleanType.cs

```diff
@@ -13,6 +14,9 @@ namespace NHibernate.Type
 	[Serializable]
 	public class BooleanType : PrimitiveType, IDiscriminatorType
 	{
+		internal static readonly object TrueObject = true; // skipcq: CS-P1003
```

Comment:
Please remove all `skipcq` comments and undo changes in `NHibernate.sln.DotSettings` file. Is it for deepsource?
Just ignore errors that you think if false positive or not caused directly by your changes.

Reply:
@bahusoid , yes, this is for DeepSource.
To me ignoring looks weird because:
- it will lead the project analysis to produce more and more noise, and contributors have to recheck the same "false positives" again and again as the change the same files again and again.
- CI checks look failed from time to time.

I'm OK to revert these changes, if it is a project convention.
Is it a convention to not silence not relevant findings in DeepSource?

Reply:
We mark important checks as required and it's not one of them. There is a convention to not modify code that is not part of your feature.

And regarding comments. My view on it - I don't want to pollute code with some deepsource specific instructions. It's just some 3d party tool we use right now and this particular issue looks like deepsource false positive detection.

Reply:
Yeah, I've got you opinion.
But is it a general opinion across the project contributors?
@hazzik , may you share your thoughts?

Reply:
changed